extras/examples/linux/README.md

```diff
@@ -387,6 +462,114 @@ Example channels configuration (details are exaplained later):
         battery_level: 2
         multiplier_battery_level: 100
 
+    # with new MQTT source
+    - type: ThermometerParsed
+      temperature: temperature
+      multiplier: 1
+      parser:
+        type: Json
+        refresh_time_ms: 200
+      source:
+        type: MQTT
+        state_topic: "sd4l/sensors/temp/0/state"
+
+    - type: ThermHygroMeterParsed
+      temperature: temperature
+      humidity: humidity
+      multiplier_temp: 1
+      multiplier_humi: 1
+      parser:
+        type: Json
+        refresh_time_ms: 200
+      source:
+        type: MQTT
+        state_topic: "sd4l/sensors/temphum/0/state"
+
+    - type: ThermHygroMeterParsed
+      name: th5
+      temperature: 0
+      humidity: 1
+      multiplier_temp: 1
+      multiplier_humi: 1
+      parser:
+        type: Simple
+        refresh_time_ms: 200
+      source:
+        type: MQTT
+        state_topic: "sd4l/sensors/temphum/2/state"
+        sub_topics: [temperature, humidity]
+
+    - type: BinaryParsed
+      state: state
+      parser:
+        type: Json
+        refresh_time_ms: 1000
+      source:
+        type: MQTT
+        state_topic: "sd4l/sensors/binary/4/state"
+
+    - type: BinaryParsed
+      state: state
+      state_on_values: ["connect"]
+      parser:
+        type: Json
+        refresh_time_ms: 1000
+      source:
+        type: MQTT
+        state_topic: "sd4l/sensors/binary/2/state"
+
+    - type: CustomRelay
+      name: custom_relay_1
+      state: state
+      set_state: 0
```

Comment:
What is "set_state" parameter for? It is not described in readme, and from code I wasn't sure how and what for it is used

Reply:
`set_state` to w `output` odpowiednik `state` w `source`

Reply:
Opisałem to pobieżnie w readme.md